pkgs/top-level/all-packages.nix

```diff
@@ -4364,6 +4364,7 @@ with pkgs;
   pepper = callPackage ../tools/admin/salt/pepper { };
 
   percona-xtrabackup = callPackage ../tools/backup/percona-xtrabackup {
+    stdenv = overrideCC stdenv gcc5;
```

Comment:
Do we really need to downgrade to gcc5? Fixing the build with gcc7 would be the preferred solution.

Reply:
I do not know how to fix an error when building with gcc7